src/drivers/imx/sdma.c

```diff
@@ -582,6 +582,8 @@ static int sdma_copy(struct dma_chan_data *channel, int bytes, uint32_t flags)
 	notifier_event(channel, NOTIFIER_ID_DMA_COPY,
 		       NOTIFIER_TARGET_CORE_LOCAL, &next, sizeof(next));
 
+	sdma_enable_channel(channel->dma, channel->index);
+
```

Comment:
Commit message has a typo on this one:

"In order, for the channel to be reschedule we need to start again the
HSTART_HE after we got an interrupt that a b_o_d was processed."

Reply:
@paulstelian97 thanks. will fix.

Reply:
Waiting for this fix, I guess I can approve after that.

Reply:
Oh, sorry forgot about this. Pushed now.